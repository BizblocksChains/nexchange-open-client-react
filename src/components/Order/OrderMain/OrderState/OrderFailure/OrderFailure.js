import React from 'react';
import config from 'Config';
import OrderLinks from '../OrderLinks/OrderLinks';
import { I18n } from 'react-i18next';

const OrderFailure = props => (
  <I18n ns="translations">
    {t => (
      <div id="order-failure" className="text-center">
        <h2>{t('order.failure1')}</h2>
        <h5>
          {t('order.failure2')} <a href={`mailto:${config.SUPPORT_EMAIL}`}>{config.SUPPORT_EMAIL}.</a>
        </h5>

        <OrderLinks {...props} />
      </div>
    )}
  </I18n>
);

export default OrderFailure;
