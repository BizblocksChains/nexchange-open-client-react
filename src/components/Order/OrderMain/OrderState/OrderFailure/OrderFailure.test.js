import React from 'react';
import { shallow } from 'enzyme';
import OrderFailure from './OrderFailure';

describe('OrderFailure', () => {
  it('renders correctly', () => {
    const wrapShallow = shallow(<OrderFailure />).dive();
    expect(wrapShallow).toMatchSnapshot();
  });
});
