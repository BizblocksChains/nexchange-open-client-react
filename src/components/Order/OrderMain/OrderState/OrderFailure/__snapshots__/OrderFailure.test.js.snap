// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderFailure renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <I18n
    ns="translations"
>
    [Function]
</I18n>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2>
          order.failure1
</h2>,
        <h5>
          order.failure2
           
          <a
                    href="mailto:support@n.exchange"
          >
                    support@n.exchange
                    .
          </a>
</h5>,
        <OrderLinks />,
      ],
      "className": "text-center",
      "id": "order-failure",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "order.failure1",
        },
        "ref": null,
        "rendered": "order.failure1",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "order.failure2",
            " ",
            <a
              href="mailto:support@n.exchange"
>
              support@n.exchange
              .
</a>,
          ],
        },
        "ref": null,
        "rendered": Array [
          "order.failure2",
          " ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "support@n.exchange",
                ".",
              ],
              "href": "mailto:support@n.exchange",
            },
            "ref": null,
            "rendered": Array [
              "support@n.exchange",
              ".",
            ],
            "type": "a",
          },
        ],
        "type": "h5",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2>
            order.failure1
</h2>,
          <h5>
            order.failure2
             
            <a
                        href="mailto:support@n.exchange"
            >
                        support@n.exchange
                        .
            </a>
</h5>,
          <OrderLinks />,
        ],
        "className": "text-center",
        "id": "order-failure",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "order.failure1",
          },
          "ref": null,
          "rendered": "order.failure1",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "order.failure2",
              " ",
              <a
                href="mailto:support@n.exchange"
>
                support@n.exchange
                .
</a>,
            ],
          },
          "ref": null,
          "rendered": Array [
            "order.failure2",
            " ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "support@n.exchange",
                  ".",
                ],
                "href": "mailto:support@n.exchange",
              },
              "ref": null,
              "rendered": Array [
                "support@n.exchange",
                ".",
              ],
              "type": "a",
            },
          ],
          "type": "h5",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
